### PR TITLE
Query series images with accept_languages to be consistent and get th…

### PR DESCRIPTION
**Pull request by medariox**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

…e expected results

### CODE REVIEW

File: medusa/indexers/tvdbv2/tvdbv2_api.py [hunk 1]
```diff
@@ -340,7 +341,8 @@ def _parse_images(self, sid):
                 if image_type not in _images:
                     _images[image_type] = {}
 
-                images = self.series_api.series_id_images_query_get(sid, key_type=image_type)
+                images = self.series_api.series_id_images_query_get(sid, key_type=image_type,
+                                                                    accept_language=self.config['language'])
```
Comment (medariox) [inline medusa/indexers/tvdbv2/tvdbv2_api.py]: This is the meat of the PR. I added an Italian show and got only some images, with the language added I get the expected images (all images available on thetvdb).

File: medusa/indexers/tvdbv2/tvdbv2_api.py [hunk 2]
```diff
@@ -331,7 +331,8 @@ def _parse_images(self, sid):
         for image_type, image_count in self._object_to_dict(series_images_count).iteritems():
             try:
                 if search_for_image_type and search_for_image_type != image_type:
-                    if image_type == 'poster' and search_for_image_type != 'poster_thumb':
+                    # We want to use the 'poster' image also for the 'poster_thumb' type
+                    if image_type != 'poster' or image_type == 'poster' and search_for_image_type != 'poster_thumb':
```
Comment (medariox) [inline medusa/indexers/tvdbv2/tvdbv2_api.py]: This logic was erroneous, my fault, sorry.

File: medusa/indexers/tvdbv2/tvdbv2_api.py (hunk 1 above)
Comment (fernandog) [inline medusa/indexers/tvdbv2/tvdbv2_api.py]: what if `images` returns Italian poster and an English poster? Which will be used? shouldn't we put meta data language first?

Comment (fernandog) [inline medusa/indexers/tvdbv2/tvdbv2_api.py]: @medariox ". I added an Italian show " - meta data language is Italian.

so are you 100% sure that if `images` has Italian poster and English poster, medusa will use the Italian (if exists) over English

Comment (medariox) [inline medusa/indexers/tvdbv2/tvdbv2_api.py]: It will only return images for the requested language.

**Comment by p0psicles** [comment]
Just to share a little how the image parsing is working.
This is where the images are retrieved from tvdb:
https://github.com/pymedusa/Medusa/blob/d2f03ff66da41b37b80a827301f530adca71b383/medusa/indexers/tvdbv2/tvdbv2_api.py#L347

Bur we save a specific image from this large list, based on how it's rated.
I thought of it as a generic method. But maybe it's tvdb specific and needs to be moved. Anyway, maybe this is causing some issues?
https://github.com/pymedusa/Medusa/blob/d2f03ff66da41b37b80a827301f530adca71b383/medusa/indexers/indexer_base.py#L209

**Comment by medariox** [comment]
@p0psicles 
Yes, but that happens after series_api.series_id_images_query_get() and I didn't notice any issue with it so far. Anyway, the issue here is that we didn't get any results from series_api.series_id_images_query_get() in the first place.

**Pull request merged**